QuestionTwo.cpp

```diff
@@ -46,6 +58,17 @@ int main()
     
     // create the linked list
     
-    node* head = new node();
-    findKToLastElement(head, k);
+    node* nodes[10];
+    
+    for(int i=1; i <=10; i++)
```

Comment:
Array indices go from 0 to n-1, so when i=10 you are accessing outside of the array...